pkg/logql/metrics.go

```diff
@@ -243,10 +242,6 @@ func RecordRangeAndInstantQueryMetrics(
 	recordUsageStats(queryType, stats)
 }
 
-func humanizeBytes(val uint64) string {
-	return strings.Replace(humanize.Bytes(val), " ", "", 1)
-}
-
```

Comment:
Basically just moved this to [utils.HumanizeBytes](https://github.com/grafana/loki/pull/14124/files#diff-c9fbe094526241a09f3baf0f1dc8da4f1ee7c25ab88ed28da5103b160860c0f4R846-R850)